docs/extras/building_vtk.rst

```diff
@@ -2,78 +2,256 @@
 
 Building VTK
 ============
-Kitware provides Python wheels for VTK at `PyPI VTK <https://pypi.org/project/vtk/>`_, but there are situations where you
+Kitware provides Python wheels for VTK at `PyPI VTK
+<https://pypi.org/project/vtk/>`_, but there are situations where you
 may need to build VTK from source (e.g. new release of Python, EGL
 rendering, additional features, etc).  As ``pyvista`` does not provide
 ``vtk``, you will have to build it manually.
 
+Should you need a prebuilt wheel, a variety of prebuilt wheels
+(including ones with EGL) can be found at `pyvista Release 0.31.0
+<https://github.com/pyvista/pyvista/releases/tag/0.31.0>`_.  This
+includes wheels for Raspberry Pi (64-bit OS), Python 3.9 Linux, and
+EGL support (``arch64`` and ``x86_64``).
+
 
 Wheels on Linux and Mac OS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Building VTK from source on Linux is fairly easy.  Using the default
-build settings, build a Python wheel of VTK using ``ninja`` using the following script.  This script assumes Python 3.9, but you can use any modern Python version.
-For some additional useful options, see the `conda-forge recipe <https://github.com/conda-forge/vtk-feedstock/blob/master/recipe/build.sh>`__.
-Most of the ones below are designed to reduce the build time and resulting wheel size.
+Building VTK from source on Linux and MacOS is fairly easy.  Using the
+default build settings, build a Python wheel of VTK using ``ninja``
+using the following script.  This script uses system python3, but you
+can use any modern Python version.  For some additional useful
+options, see the `conda-forge recipe
+<https://github.com/conda-forge/vtk-feedstock/blob/master/recipe/build.sh>`__.
+Most of the ones below are designed to reduce the build time and
+resulting wheel size.
 
 .. code-block:: bash
 
+    #!/bin/bash
+
+    # install build dependencies (Linux/Debian)
+    sudo apt-get install ninja-build cmake libgl1-mesa-dev python3-dev -y
+
+    # Linux/CentOS
+    sudo yum install epel-release -y
+    sudo yum install ninja-build cmake mesa-libGL-devel mesa-libGLU-devel -y
```

Comment:
Is it not dangerous to recommend `sudo apt-get install -y`? I've had situations where installing `python3-dev` would have wanted to uninstall my entire gnome (admittedly on an older debian installation with many subsequent upgrades and manual package manipulations).

I'd recommend letting the user read the instructions and choose -- because this first script would not typically be run in docker containers.

Reply:
I agree with this.  When I wrote this, I copied all the work I had done on my end over to the docs and didn't think to remove `-y`.  I agree that it's a security/stability concern.

Additionally, docker will automatically add `-y`, so there's no reason to include it at all.